doc/guides/build/kconfig/official_doc.rst

```diff
@@ -0,0 +1,11 @@
+.. _kconfig_official_doc:
+
+Kconfig - Official Documentation
+################################
+
+.. note::
+
+   The official Kconfig documentation is `kconfig-language.rst
+   <https://www.kernel.org/doc/html/latest/kbuild/kconfig-language.html>`__
+   and `kconfig-macro-language.rst
+   <https://www.kernel.org/doc/html/latest/kbuild/kconfig-macro-language.html>`__.
```

Comment:
I'd personally not create an extra page for just a couple of links. Some description with these links in `Configuration System (Kconfig)` section should be enough to provide better visibility.

Reply:
Better than now, for sure.  But not as good as getting them into the table of contents.